nim/game.py

```diff
@@ -92,6 +95,18 @@ def __init__(self):
         self.chaos = Chaos()
 
     def new_game(self, min, max, piles):
+        """
+        Logic of new game dealt with here.
+        Uses either user-passed JSON or
+        default values to fill in params
```

Comment:
JSON? Default values?

Reply:
I meant that the route is either passing parameters from JSON we get from user action or the default values in `DEFAULT_GAME`

Reply:
Except that it isn't. All that `new_game()` knows about is `min`, `max` and `piles`. Here we know nothing about where that data came from. Sure right now our only implementation follows what you said in the comment, but that's only one implementation. That comment doesn't belong here because `new_game()` doesn't know about this at all.

Reply:
That's what I had been trying not to say for days so that Dave would discover it on his own, now it's all spoiled :stuck_out_tongue:

Reply:
Haha, sorry! >_< I'll let you do your thing :)